services/app-api/forms/mcpar.json

```
@@ -1066,7 +1058,7 @@
                 "validation": "text",
                 "props": {
                   "label": "C1.III.5 Incentives for encounter data quality",
-                  "hint": "Describe the types of incentives that may be awarded to managed care plans for encounter data quality. Reply with \"N/A\" if the plan does not use incentives to award encounter data quality."
+                  "hint": "Describe the types of incentives that may be awarded to managed care plans for encounter data quality. Reply with “N/A“ if the plan does not use incentives to award encounter data quality."
```

fix double open quote
```suggestion
                  "hint": "Describe the types of incentives that may be awarded to managed care plans for encounter data quality. Reply with “N/A” if the plan does not use incentives to award encounter data quality."
```